common/models/UserLocalSettings.ts

```diff
@@ -7,6 +7,7 @@ export type UserLocalSettings = {
     deviceInfo?: DeviceInfo;
 
     notifications?: NotificationsSettings;
+    health?: HealthPermissionsSettings;
```

Comment:
Better to keep names consistent, so I would name this `healthPermissions` rather than `health`
```suggestion
    healthPermissions?: HealthPermissionsSettings;
```